osbuild-composer.spec

```
@@ -67,6 +67,12 @@ Obsoletes: lorax-composer <= 29
 Conflicts: lorax-composer
 %endif
 
+# Remove when we stop releasing into Fedora 35
+%if 0%{?fedora} >= 34
+# lorax 34.3 is the first one without the composer subpackage
+Obsoletes: lorax-composer < 34.3
```

This way, you can easily reintroduce it if needed if this ends up in the Fedora 34 GA repo.

```suggestion
Obsoletes: lorax-composer < 34.3-100
```

If you are 100 % positive you won't even need to reintroduce lorax-composer, you can ignore this suggestion.

---

lorax-composer is gone, the code has been removed from master and rawhide and will not be returning.